ffv1.md

```diff
@@ -53,7 +53,7 @@ The FFV1 bitstream is described in this document using pseudo-code. Note that th
 
 ### Arithmetic Operators
 
-Note: the operators and the order of precedence are the same as used in the C programming language [@!ISO.9899.2018].
+Note: the operators and the order of precedence are the same as used in the C programming language [@!ISO.9899.2018]. With the exception of `>>` (removial of implementation defined behavior) and `^` (power instead of XOR)
```

Comment:
Suggested rewording.
> Note: the operators and the order of precedence are the same as used in the C programming language [@!ISO.9899.2018], with the exception of `a >> b` and `a ^ b` operators which are re-defined within this section.

Reply:
I think the wording should still contain a summary of the differences. This makes the text easier to understand. And not just correct

Reply:
I'd be fine with adding the parenthetical hints back in. My initial motivation to edit was to merge the two sentences and correct a typo ('removial').

Reply:
would that work: ?
"With the exception of `>>` (removal of implementation defined behavior) and `^` (power instead of XOR) operators which are re-defined within this section."

Reply:
+1